package.json

```diff
@@ -17,7 +17,7 @@
     "jquery-extendext": "^0.1.2",
     "jquery-ui-bundle": "^1.11.4",
     "moment": "^2.17.1",
-    "reconcile.js": "github:matthewma7/reconcile.js"
+    "reconcile.js": "^1.3.2"
```

Comment:
👍

Reply:
@aashish24 Yes, I contacted the author and merged in the feature I needed.